examples/deployAccount/main.go

```diff
@@ -64,38 +59,56 @@ func main() {
 	}
 
 	// Create transaction data
-	tx := rpc.DeployAccountTxn{
-		Nonce:               &felt.Zero, // Contract accounts start with nonce zero.
-		MaxFee:              new(felt.Felt).SetUint64(4724395326064),
-		Type:                rpc.TransactionType_DeployAccount,
-		Version:             rpc.TransactionV1,
-		Signature:           []*felt.Felt{},
-		ClassHash:           classHash,
-		ContractAddressSalt: pub,
-		ConstructorCalldata: []*felt.Felt{pub},
+	tx := rpc.BroadcastDeployAccountTxn{
+		DeployAccountTxn: rpc.DeployAccountTxn{
+			Nonce:               &felt.Zero, // Contract accounts start with nonce zero.
+			MaxFee:              new(felt.Felt).SetUint64(7268996239700),
```

Comment:
I had to increase the max fee to 117489367724301, in order for this to work.

Reply:
This was due to high gas price volatility. I fixed this in the new commit by updating the transaction's max fee field with the value returned by EstimateFee + 20%